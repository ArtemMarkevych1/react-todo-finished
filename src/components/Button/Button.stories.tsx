import React from 'react'
import { storiesOf } from '@storybook/react'
import { APP_NAME } from '../../utils'

import Button from './Button'

storiesOf(APP_NAME, module).add('Button', () => <Button>{APP_NAME}</Button>)
