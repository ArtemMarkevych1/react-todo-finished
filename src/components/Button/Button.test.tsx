import React from 'react'
import { shallow } from 'enzyme'
import Button from './Button'

describe('Button', () => {
  it('should look like init snapshot', () => {
    const component = shallow(<Button />)

    expect(component).toMatchSnapshot()
  })
})
