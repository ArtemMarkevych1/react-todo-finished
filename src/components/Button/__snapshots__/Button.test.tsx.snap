// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should look like init snapshot 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  background-color: rgb(255, 255, 255);
  border: 4px;
  border-radius: 0px;
  color: #4d4d4d;
  font: 14px 'Helvetica Neue', Helvetica, Arial, sans-serif;
  padding: 6px 12px;
  font-weight: 300;
  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.2), 0 25px 50px 0 rgba(0, 0, 0, 0.1);
",
        ],
      },
      "displayName": "styled.button",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "button",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
